### PR TITLE
*: Add grafana for PD http api request duration

### LINKED ISSUE #33916: Add grafana chart for pd http request duration
## Enhancement

As the states in the title

### PR DESCRIPTION
**Pull request by lcwangchao**
### What problem does this PR solve?

Issue Number: close #33916

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
Add a chart in grafana to view PD http api duration
```

### CODE REVIEW

File: metrics/grafana/tidb.json [hunk 1]
```diff
@@ -8802,6 +8802,129 @@
...
             "alignLevel": null
           }
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_TEST-CLUSTER}",
+          "description": "pd http request durations by type within 99.9 percent buckets",
```
Comment (lcwangchao) [inline metrics/grafana/tidb.json:8812]:
```suggestion
          "description": "pd http request durations",
```

File: metrics/grafana/tidb.json [hunk 2]
```diff
@@ -8802,6 +8802,129 @@
...
+              "legendFormat": "99-{{type}}",
+              "refId": "B"
+            },
+            {
+              "exemplar": true,
+              "expr": "histogram_quantile(0.90, sum(rate(tidb_server_pd_api_execution_duration_seconds_bucket{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}[1m])) by (le, type))",
+              "format": "time_series",
+              "interval": "",
+              "intervalFactor": 2,
+              "legendFormat": "90-{{type}}",
```
Comment (hawkingrei) [inline metrics/grafana/tidb.json:8883]: it should call ```P90```, ```Ppp```?

Comment (lcwangchao) [reply metrics/grafana/tidb.json:8883]: This is copied from other charts to keep a consistency.

**Review by wjhuang2016 — APPROVED** [review]

File: metrics/grafana/tidb.json (hunk 2 above)
Comment (hawkingrei) [reply metrics/grafana/tidb.json:8883]: OK

**Review by hawkingrei — APPROVED** [review]

**Comment by purelind** [comment]
### TiDB MergeCI notify
**✅ Well Done! New fixed [1] after this pr merged.**
CI Name | Result | Duration | Compare with Parent commit
------------ | ------------- | ------------- | -------------
[idc-jenkins-ci-tidb/integration-copr-test](https://ci.pingcap.net/job/tidb_ghpr_integration_copr_test/9053/display/redirect) | ✅ all 1 tests passed | 4 min 28 sec | Fixed
[idc-jenkins-ci/integration-cdc-test](https://ci.pingcap.net/job/tidb_ghpr_integration_cdc_test/1817/display/redirect) | 🟢 all 34 tests passed | 37 min | Existing passed
[idc-jenkins-ci-tidb/integration-br-test](https://ci.pingcap.net/job/tidb_ghpr_integration_br_test/8789/display/redirect) | 🟢 all 29 tests passed | 28 min | Existing passed
[idc-jenkins-ci-tidb/code-coverage](https://ci.pingcap.net/job/tidb_ghpr_coverage/3004/display/redirect) | 🟢 Lines coverage: 63.49% | 13 min | Existing passed
[idc-jenkins-ci-tidb/integration-common-test](https://ci.pingcap.net/job/tidb_ghpr_integration_common_test/10644/display/redirect) | 🟢 all 11 tests passed | 9 min 55 sec | Existing passed
[idc-jenkins-ci-tidb/sqllogic-test-1](https://ci.pingcap.net/job/tidb_ghpr_sqllogic_test_1/9201/display/redirect) | 🟢 all 26 tests passed | 7 min 20 sec | Existing passed
[idc-jenkins-ci-tidb/common-test](https://ci.pingcap.net/job/tidb_ghpr_common_test/9475/display/redirect) | 🟢 all 12 tests passed | 7 min 12 sec | Existing passed
[idc-jenkins-ci-tidb/integration-ddl-test](https://ci.pingcap.net/job/tidb_ghpr_integration_ddl_test/9264/display/redirect) | 🟢 all 6 tests passed | 6 min 50 sec | Existing passed
[idc-jenkins-ci-tidb/tics-test](https://ci.pingcap.net/job/tidb_ghpr_tics_test/8893/display/redirect) | 🟢 all 1 tests passed | 6 min 10 sec | Existing passed
[idc-jenkins-ci-tidb/sqllogic-test-2](https://ci.pingcap.net/job/tidb_ghpr_sqllogic_test_2/9139/display/redirect) | 🟢 all 28 tests passed | 6 min 8 sec | Existing passed
[idc-jenkins-ci-tidb/mybatis-test](https://ci.pingcap.net/job/tidb_ghpr_mybatis/9063/display/redirect) | 🟢 all 1 tests passed | 3 min 44 sec | Existing passed
[idc-jenkins-ci-tidb/integration-compatibility-test](https://ci.pingcap.net/job/tidb_ghpr_integration_campatibility_test/10051/display/redirect) | 🟢 all 1 tests passed | 3 min 24 sec | Existing passed
[idc-jenkins-ci-tidb/plugin-test](https://ci.pingcap.net/job/tidb_merged_pr_build/2873/display/redirect) | 🟢 build success, plugin test success | 4min | Existing passed

**Pull request merged**